pkgs/applications/office/scribus/unstable.nix

```diff
@@ -1,26 +1,24 @@
-{ stdenv, fetchsvn, wrapQtAppsHook, pkgconfig, cmake, qtbase, cairo, pixman,
+{ stdenv, fetchurl, wrapQtAppsHook, pkgconfig, cmake, qtbase, cairo, pixman,
 boost, cups, fontconfig, freetype, hunspell, libjpeg, libtiff, libxml2, lcms2,
 podofo, poppler, poppler_data, python2, harfbuzz, qtimageformats, qttools }:
 
 let
   pythonEnv = python2.withPackages(ps: [ps.tkinter ps.pillow]);
-  revision = "22806";
 in
 stdenv.mkDerivation rec {
-  name = "scribus-unstable-${version}";
-  version = "2019-01-16";
```

Comment:
It actually appears this  is a downgrade according to the release date
*  2018-04-28

> https://bugs.scribus.net/changelog_page.php?version_id=106

Reply:
That's weird. I got the link to the change log from this recent [release announcement on their mailing list](http://lists.scribus.net/pipermail/scribus/2019-August/055703.html). I guess the date 2018-04-28 means, that the changelog was released on the last release 1.5.4 and got appended until recently.
I also checked the checksum in the announcement mail and they match with the tar ball introduced by this PR.